translator/translate/otel/processor/resourcedetection/translator.go

```diff
@@ -64,6 +64,7 @@ func (t *translator) ID() component.ID {
 func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
 	cfg := t.factory.CreateDefaultConfig().(*resourcedetectionprocessor.Config)
 
+	//cfg.MiddlewareID = &agenthealth.StatusCodeID  This is going to be added when contrib changes is made
```

Comment:
instead of that comment can you also add a warning in your PR description stating you need that specific contrib PR: 

https://github.com/orgs/community/discussions/16925#discussion-4085374

Reply:
done